programs/Python/calculator.py

```diff
@@ -0,0 +1,15 @@
+#Program for calculator in python
+a=int(input("Enter first no.- "))
+b=int(input("Enter second no.- "))
+operator=input("Enter the operation you want to perform ")
+if(operator == "+"):
```

Comment:
space before "(" in all the subsequent if or elif statements.

Reply:
Just that, you want spaces there?